catboost/python-package/catboost/widget/callbacks.py

```diff
@@ -0,0 +1,101 @@
+import xgboost as xgb
```

Comment:
Please make this import optional to make catboost widget independent of xgboost installation.